components/11-argo-workflow/base/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# Source: nautobot/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argo-configmap
+  namespace: "argo"
+  labels:
+    app.kubernetes.io/name: argo-workflows
+    helm.sh/chart: argo-workflows-0.40.10
+    app.kubernetes.io/instance: argo
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/version: "0.40.10"
```

Comment:
The hardcoded versions here will break in the future. But what's the point of the ConfigMap? it's empty?

Reply:
there is no point.. it was a place holder but I can remove it